### PR TITLE
Update zh_CN.json

### PR DESCRIPTION
**Pull request by zilinwsx**

### CODE REVIEW

**Review by Yimura — COMMENTED** [review]
I can't curate the quality of these translations, @sch-lda would you be willing to have a look on if these translations are any good?

File: zh_CN.json [hunk 1]
```diff
@@ -28,32 +28,32 @@
...
   "BREAKUP_KICK": "分离踢",
-  "BRING_ALL_DESC": "传送所有玩家到你,即使他们不在载具中",
-  "BRING_ALL": "带来全部",
+  "BRING_ALL_DESC": "传送所有玩家到你,不必在载具中",
+  "BRING_ALL": "带来全部玩家",
   "BRING_CLOSEST_VEHICLE": "进入附近的载具",
-  "BRING_DESC": "传送玩家到你的当前位置",
-  "BRING_PV": "传送个人载具到身边",
+  "BRING_DESC": "传送玩家到你的位置",
+  "BRING_PV": "召唤个人载具(技工)",
```
Comment (sch-lda) [inline zh_CN.json:43]: 技工运送个人载具 or omit brackets

File: zh_CN.json [hunk 2]
```diff
@@ -28,32 +28,32 @@
...
   "BRING": "带来",
   "BULLET_IMPACT": "子弹效果",
   "BULLET": "子弹",
   "BULLETPROOF_TIRES": "防弹轮胎",
   "BUSINESS_BATTLES": "商战",
-  "BYPASS_C4_LIM": "绕过C4数量限制",
+  "BYPASS_C4_LIM": "提升C4数量限制",
```
Comment (sch-lda) [inline zh_CN.json:49]: 绕过-限制 or 提升-上限

File: zh_CN.json [hunk 3]
```diff
@@ -28,32 +28,32 @@
...
   "CAMHEDZ": "猎头",
   "CANCEL": "取消",
   "CAPTURE": "占领",
   "CEO_ABILITIES": "CEO能力",
   "CEO_BULLSHARK": "牛鲨睾酮",
-  "CEO_KICK_DESC": "踢出组织或摩托帮成员，对CEO或首领无效",
-  "CEO_KICK": "CEO踢",
+  "CEO_KICK_DESC": "取消副手身份，对CEO或首领无效",
```
Comment (sch-lda) [inline zh_CN.json:55]: 副手 cannot represent both CEO and MC

File: zh_CN.json [hunk 4]
```diff
@@ -28,32 +28,32 @@
...
+  "CEO_KICK": "取消CEO身份",
```
Comment (sch-lda) [inline zh_CN.json:56]: There's no need to change that

File: zh_CN.json [hunk 5]
```diff
@@ -73,13 +73,13 @@
   "CLOSE_ALL_DOORS": "全关",
   "CLOSE": "关",
   "CMD_EXECUTOR_CMD_TEMPLATE": "{} - {} - {} | 需要 {} 个参数",
-  "CMD_EXECUTOR_MULTIPLE_CMDS": "你可以同时执行几个命令，用 \";\" 来分隔它们",
+  "CMD_EXECUTOR_MULTIPLE_CMDS": "可以同时执行多条命令，用 \";\" 分隔每一条命令",
```
Comment (sch-lda) [inline zh_CN.json:76]: Why omit the subject

File: zh_CN.json [hunk 6]
```diff
@@ -135,7 +135,7 @@
   "DEBUG_SCRIPTS_SUB_TITLE": "这些脚本负责所有循环功能。\n只有当你知道自己在做什么时才去禁用",
   "DEBUG_TAB_GLOBALS": "全局",
   "DEBUG_TAB_LOCALS": "本地",
-  "DEBUG_TAB_MISC": "杂项",
+  "DEBUG_TAB_MISC": "常用",
```
Comment (sch-lda) [inline zh_CN.json:138]: There's no need to change that.

File: zh_CN.json [hunk 7]
```diff
@@ -215,8 +215,8 @@
   "FORCE_WANTED_LVL_INFO": "强制自己获得一个特定的通缉等级",
   "FORCE_WANTED_LVL": "强制通缉等级",
   "FORCE": "强制",
-  "FREECAM_DESC": "将镜头与玩家分离,你将能够自由移动镜头",
-  "FREECAM": "自由镜头",
+  "FREECAM_DESC": "将镜头与玩家分离,你将能够自由移动镜头,像无人机一般",
+  "FREECAM": "无人机镜头",
```
Comment (sch-lda) [inline zh_CN.json:219]: There's no need to change that.

File: zh_CN.json [hunk 8]
```diff
@@ -234,31 +234,31 @@
   "GAME_CACHE_WAITING_FOR_SINGLE_PLAYER": "等待进入故事模式更新缓存...",
   "GAME_CACHE": "游戏缓存",
   "GENERAL": "常规",
-  "GET_ALL_WEAPONS": "获得所有武器",
+  "GET_ALL_WEAPONS": "获得所有武器(不会显示在武器库)",
```
Comment (sch-lda) [inline zh_CN.json:237]: There is no need to explain these common features

File: zh_CN.json [hunk 9]
```diff
@@ -234,31 +234,31 @@
...
   "GIVE_ARMOR": "给予防弹衣",
   "GIVE_HEALTH_DESC": "给予玩家一些急救包",
   "GIVE_HEALTH": "恢复生命值",
-  "GIVE_WEAPONS_ALL_DESC": "给予所有玩家所有武器.快速多次按下将导致自身游戏崩溃",
+  "GIVE_WEAPONS_ALL_DESC": "给予所有玩家所有武器.快速多次按下将导致游戏崩溃",
   "GIVE_WEAPONS_DESC": "给予玩家所有武器",
   "GIVE_WEAPONS": "给予武器",
   "GOD_MODE": "载具无敌",
-  "GODMODE_DESC": "让您免受任何伤害",
+  "GODMODE_DESC": "让您免受任何伤害,不适用于无效边界死亡",
```
Comment (sch-lda) [inline zh_CN.json:248]: There is no need to explain these common features

File: zh_CN.json [hunk 10]
```diff
@@ -234,31 +234,31 @@
...
   "GUI_TAB_CREATOR": "制作器",
   "GUI_TAB_DEBUG": "调试",
-  "GUI_TAB_ESP": "ESP",
+  "GUI_TAB_ESP": "ESP(透视)",
   "GUI_TAB_GUI": "GUI",
   "GUI_TAB_HANDLING_CURRENT_PROFILE": "当前配置文件",
   "GUI_TAB_HANDLING_SAVED_PROFILE": "保存配置文件",
   "GUI_TAB_HANDLING": "操控性",
-  "GUI_TAB_HOTKEYS": "热键",
+  "GUI_TAB_HOTKEYS": "按键自定义",
```
Comment (sch-lda) [inline zh_CN.json:261]: There's no need to change that.

File: zh_CN.json [hunk 11]
```diff
@@ -272,11 +272,11 @@
   "GUI_TAB_OUTFIT_SLOTS": "保存的套装",
   "GUI_TAB_PERSIST_CAR": "保存载具",
   "GUI_TAB_PERSONAL_VEHICLE": "个人载具",
-  "GUI_TAB_PLAYER_DB": "玩家数据库",
+  "GUI_TAB_PLAYER_DB": "作弊玩家数据库",
```
Comment (sch-lda) [inline zh_CN.json:275]: Player database may not only include cheaters.

File: zh_CN.json [hunk 12]
```diff
@@ -272,11 +272,11 @@
...
   "GUI_TAB_PROTECTION": "防护",
-  "GUI_TAB_REACTIONS": "反应",
-  "GUI_TAB_SELF": "自身选项",
-  "GUI_TAB_SESSION_BROWSER": "战局浏览器",
+  "GUI_TAB_REACTIONS": "日志与广播",
```
Comment (sch-lda) [inline zh_CN.json:277]: 反应 contains 日志与广播/Kick/Add to DB/Block join. But 日志与广播 do not contains Kick/Add to DB/Block join

File: zh_CN.json [hunk 13]
```diff
@@ -272,11 +272,11 @@
...
+  "GUI_TAB_SELF": "个人选项",
+  "GUI_TAB_SESSION_BROWSER": "战局筛选器",
```
Comment (sch-lda) [inline zh_CN.json:279]: Strange way of expression

File: zh_CN.json [hunk 14]
```diff
@@ -608,7 +608,7 @@
   "PLAYER_OPTIONS": "玩家选项: {}",
   "PLAYER_SPOOFED_NAME": "名称: ",
   "PLAYER_SWITCH_IN_PROGRESS": "正处于战局过渡状态，请稍等",
-  "PLAYER_TOXIC_BRING_PLAYER_OUT_GOD": "如果攻击没有被阻止，对方的无敌将失效",
+  "PLAYER_TOXIC_BRING_PLAYER_OUT_GOD": "如果攻击成功，对方将(可能)失去无敌状态",
```
Comment (sch-lda) [inline zh_CN.json:611]: There's no need to change that.

File: zh_CN.json [hunk 15]
```diff
@@ -627,13 +627,13 @@
   "PRIMARY": "主色调",
   "PROOFS": "保护",
   "PROTECTIONS": "恶意行为防护",
-  "PTFX_CMD_DESC": "在你的人物和载具上显示漂亮的PTFX效果",
-  "PTFX_CMD": "启用PTFX效果",
+  "PTFX_CMD_DESC": "在人物和载具上显示漂亮的粒子效果",
+  "PTFX_CMD": "启用PTFX粒子效果",
```
Comment (sch-lda) [inline zh_CN.json:631]: 启用粒子效果 or 启用PTFX

File: zh_CN.json [hunk 16]
```diff
@@ -648,33 +648,33 @@
...
-  "RELOAD_PLYR_ONLINE_STATES": "刷新玩家在线状态",
+  "RELOAD_PLYR_ONLINE_STATES": "更新玩家在线状态",
   "REMOTE_CONTROL": "远程控制",
   "REMOTE_CRASH_INVALID_BLACKLIST_FROM": "已阻止来自 {} 的远程无效黑名单崩溃 {}",
   "REMOTE_CRASH_INVALID_BLACKLIST": "已阻止远程无效黑名单崩溃",
   "REMOTE_CRASH_INVALID_TRANSITION_FROM": "已阻止来自 {} 的无效的启动战局远程崩溃 {}",
   "REMOTE_CRASH_INVALID_TRANSITION": "已阻止无效的启动战局远程崩溃",
   "REMOTE_KICK_BREAKUP_PLAYER": "某人尝试远程分离踢出 {} !",
-  "REMOTE_KICK_BREAKUP": "某人试图对您使用远程分离踢!",
+  "REMOTE_KICK_BREAKUP": "某人试图对您使用分离踢!",
```
Comment (sch-lda) [inline zh_CN.json:658]: Why remove "Remote"

File: zh_CN.json [hunk 17]
```diff
@@ -697,12 +697,12 @@
   "SECONDARY": "副色调",
   "SELECT_FILE_FIRST": "先选择一个文件",
   "SELF": "自身",
-  "SEMI_GODMODE": "半无敌",
+  "SEMI_GODMODE": "半无敌(非致命伤保护)",
```
Comment (sch-lda) [inline zh_CN.json:700]: There is no need to explain these common features

File: zh_CN.json [hunk 18]
```diff
@@ -697,12 +697,12 @@
...
   "SEND_MSG": "发送信息",
-  "SEND_SEXT_DESC": "给玩家发送带有图片的随机内容短信",
+  "SEND_SEXT_DESC": "发送带图片的随机短信",
   "SEND_SEXT": "发送信息",
   "SEND_SMS": "发送信息",
-  "SEND_TO_CUTSCENE": "发送到过场动画",
+  "SEND_TO_CUTSCENE": "触发过场动画",
```
Comment (sch-lda) [inline zh_CN.json:705]: There's no need to change that.

File: zh_CN.json [hunk 19]
```diff
@@ -723,7 +723,7 @@
   "SETTINGS_CONTEXT_MENU_ENTITY_TYPE_PLAYER": "玩家",
   "SETTINGS_CONTEXT_MENU_ENTITY_TYPE_VEHICLE": "载具",
   "SETTINGS_CONTEXT_MENU_ENTITY_TYPES": "允许的实体类型:",
-  "SETTINGS_CONTEXT_MENU": "启用X键菜单",
+  "SETTINGS_CONTEXT_MENU": "X键实体信息菜单",
```
Comment (sch-lda) [inline zh_CN.json:726]: X键实体控制菜单/X键菜单/快捷菜单

File: zh_CN.json [hunk 20]
```diff
@@ -735,20 +735,20 @@
   "SETTINGS_ESP_FRIENDLY_COLOR": "好友颜色:",
   "SETTINGS_ESP_GLOBAL_RENDER_DISTANCE": "全局渲染距离 (最小, 最大)",
   "SETTINGS_ESP_HIDE_SELF": "隐藏自身",
-  "SETTINGS_ESP_PLAYER_ARMOR": "显示玩家护甲值",
+  "SETTINGS_ESP_PLAYER_ARMOR": "显示玩家防弹衣值",
   "SETTINGS_ESP_PLAYER_DISTANCE": "显示玩家距离",
-  "SETTINGS_ESP_PLAYER_GOD_MODE": "显示玩家无敌",
+  "SETTINGS_ESP_PLAYER_GOD_MODE": "显示玩家防护状态(无敌.防弹)",
```
Comment (sch-lda) [inline zh_CN.json:740]: There is no need to add brackets

File: zh_CN.json [hunk 21]
```diff
@@ -735,20 +735,20 @@
...
-  "SETTINGS_ESP_SCALE_ARMOR": "护甲值应该随距离缩放",
-  "SETTINGS_ESP_SCALE_HEALTH": "健康栏是否应随距离缩放",
-  "SETTINGS_ESP_TRACER_POSITION": "跟踪者绘制位置 (x, y)",
-  "SETTINGS_ESP_TRACER_RENDER_DISTANCE": "追踪者渲染距离 (min, max)",
-  "SETTINGS_ESP_TRACER": "跟踪者",
-  "SETTINGS_ESP": "启用 ESP",
+  "SETTINGS_ESP_PLAYER_NAME": "显示玩家名称",
+  "SETTINGS_ESP_SCALE_ARMOR": "防弹衣值随距离缩放(对齐方框)",
+  "SETTINGS_ESP_SCALE_HEALTH": "血量随距离缩放(对齐方框)",
+  "SETTINGS_ESP_TRACER_POSITION": "透视绘制位置 (x, y)",
```
Comment (sch-lda) [inline zh_CN.json:745]: 透视 --> 透视线

File: zh_CN.json [hunk 22]
```diff
@@ -735,20 +735,20 @@
...
+  "SETTINGS_ESP_TRACER_RENDER_DISTANCE": "射线渲染距离 (min, max)",
```
Comment (sch-lda) [inline zh_CN.json:746]: 射线 --> 透视线

File: zh_CN.json [hunk 23]
```diff
@@ -735,20 +735,20 @@
...
+  "SETTINGS_ESP_TRACER": "射线",
+  "SETTINGS_ESP": "启用ESP(透视)",
   "SETTINGS_LANGUAGES": "语言:",
-  "SETTINGS_MISC_DEV_DLC": "启用开发中的 DLC",
-  "SETTINGS_MISC": "杂项",
+  "SETTINGS_MISC_DEV_DLC": "启用未发布的DLC",
+  "SETTINGS_MISC": "常用",
```
Comment (sch-lda) [inline zh_CN.json:751]: There's no need to change that.

File: zh_CN.json [hunk 24]
```diff
@@ -757,14 +757,14 @@
   "SETTINGS_NOTIFY_MISMATCH_SYNC_TYPE": "同步类型不匹配",
   "SETTINGS_NOTIFY_OTHER": "其他",
   "SETTINGS_NOTIFY_OUT_OF_ALLOWED_RANGE_SYNC_TYPE": "超出允许范围的同步类型",
-  "SETTINGS_NOTIFY_PLAYER_JOIN_ABOVE_MAP": "地图上方",
+  "SETTINGS_NOTIFY_PLAYER_JOIN_ABOVE_MAP": "地图上方(如同游戏通知)",
```
Comment (sch-lda) [inline zh_CN.json:760]: There is no need to explain these common features

File: zh_CN.json [hunk 25]
```diff
@@ -774,7 +774,7 @@
   "SHOW_TRANSACTION_ERROR_DESC": "玩家将看到全屏显示的库存或价格无效导致交易失败的警告",
   "SHOW_TRANSACTION_ERROR": "显示交易失败警报",
   "SHOW": "显示",
-  "SKIP_CUTSCENE": "跳过过场动画",
+  "SKIP_CUTSCENE": "跳过过场动画(部分)",
```
Comment (sch-lda) [inline zh_CN.json:777]: There is no need to explain these common features

File: zh_CN.json [hunk 26]
```diff
@@ -783,7 +783,7 @@
   "SPACE_MONKEY": "太空猴游戏",
   "SPASM": "抽搐",
   "SPAWN_BODYGUARD": "生成保镖",
-  "SPAWN_CLONE": "生成克隆",
+  "SPAWN_CLONE": "生成复杂",
```
Comment (sch-lda) [inline zh_CN.json:786]: typo

File: zh_CN.json [hunk 27]
```diff
@@ -833,42 +833,42 @@
...
   "STOP_TIME": "停止时间",
   "STYLE": "风格",
   "SUPER_RUN_DESC": "让你跑得很快",
-  "SUPER_RUN": "超级跑",
+  "SUPER_RUN": "快速奔跑",
   "SUPERMAN_DESC": "像超人一样飞行",
   "SUPERMAN": "超人模式",
   "TELEPORT_DESC": "传送到玩家",
-  "TELEPORT_INTO_VEHICLE_DESC": "传送进玩家的载具.提前观看以取得理想效果",
+  "TELEPORT_INTO_VEHICLE_DESC": "进入该玩家载具.可能需要观看玩家生效",
```
Comment (sch-lda) [inline zh_CN.json:844]: There's no need to change that.

File: zh_CN.json [hunk 28]
```diff
@@ -833,42 +833,42 @@
...
+  "TP_ALL_TO_FLIGHT_SCHOOL": "所有玩家传送到飞行学校(机场)",
+  "TP_ALL_TO_FREAKSHOP": "所有玩家传送到怪胎店",
+  "TP_ALL_TO_MAP_CENTER": "所有玩家传送到地图中心",
+  "TP_ALL_TO_MOC": "所有玩家传送到机动作战中心",
+  "TP_ALL_TO_MULTI_FLOOR_GARAGE": "所有玩家传送到多层车库",
+  "TP_ALL_TO_MUSIC_LOCKER": "所有玩家传送到音乐柜",
+  "TP_ALL_TO_PENTHOUSE": "所有玩家传送到空中别墅",
+  "TP_ALL_TO_RECORD_A_STUDIOS": "所有玩家传送到录音A工作室",
+  "TP_ALL_TO_SKYDIVE": "所有玩家传送到跳伞位置",
+  "TP_ALL_TO_WAREHOUSE": "所有玩家传送到仓库",
```
Comment (sch-lda) [inline zh_CN.json:869]: 所有玩家传送到 -->传送所有玩家到

File: zh_CN.json [hunk 29]
```diff
@@ -885,24 +885,24 @@
...
+  "TROLL": "恶搞功能",
   "TURBO": "涡轮增压",
   "TURN_ENGINE_OFF": "关闭引擎",
   "TURN_ENGINE_ON": "打开引擎",
-  "TURN_INTO_BEAST_ALL_DESC": "将每一个人变成野兽,包括自己",
-  "TURN_INTO_BEAST_ALL": "将每一个人变成野兽",
-  "TURN_INTO_BEAST_DESC": "将玩家变成野兽",
+  "TURN_INTO_BEAST_ALL_DESC": "所有玩家变成野兽.包括自己",
+  "TURN_INTO_BEAST_ALL": "所有玩家变成野兽",
+  "TURN_INTO_BEAST_DESC": "所有玩家变成野兽",
```
Comment (sch-lda) [inline zh_CN.json:894]: 所有玩家变成野兽 --> 将所有玩家变成野兽

File: zh_CN.json [hunk 30]
```diff
@@ -933,14 +933,14 @@
   "VEHICLE": "载具",
   "VEHICLES": "载具:",
   "VIEW_PLAYER_PLAYER_OPTIONS": "玩家选项: %s",
-  "WALK_UNDERWATER_DESC": "允许你在水下行走甚至射击",
+  "WALK_UNDERWATER_DESC": "如同在陆地",
   "WALK_UNDERWATER": "水下行走",
-  "WANDER": "闲逛",
+  "WANDER": "闲逛(npc驾驶)",
```
Comment (sch-lda) [inline zh_CN.json:938]: There is no need to explain these common features

**Review by sch-lda — COMMENTED** [review]
The original translation does need improvement, but I don't think it needs that much.

**Comment by Yimura** [comment]
Closing this PR as it has gone stale.

@sch-lda, do you think it's worth it to update the Chinese translations? If so then if you would could you update these?

**Comment by zilinwsx** [comment]
Do you mean that there are more English translation items for me to do?If
yes.Then Id love to

Andreas Maerten ***@***.***> 于 2023年9月27日周三 上午4:12写道：

**Comment by Yimura** [comment]
Hi there @zilinwsx, I'm happy reopening this PR if you could address the comments from @sch-lda.

**Comment by sch-lda** [comment]
> Closing this PR as it has gone stale.
> 
> @sch-lda, do you think it's worth it to update the Chinese translations? If so then if you would could you update these?

I am the current yimmenu localizer in Crazyzhang's localization group. The latest zh_CN.json can always be found at https://github.com/CrazyZhang666/GTA5OnlineTools/blob/main/GTA5Shared/Files/YimMenu/zh_CN.json. I will synchronize it back to the official yimmenu repository when there are a lot of changes, but of course, anytime. I will make a few changes to it and submit a pull request when I'm done with today's lesson.

**Pull request closed without merging**